src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php

```diff
@@ -140,6 +151,7 @@ public function jsonSerialize()
             'status'           => (string) $this->status,
             'identity_id'      => $this->identityId,
             'name'             => $this->name,
+            'document_number'  => (string) $this->documentNumber,
```

Comment:
And I don't know why we didn't make SecondFactorStatus JsonSerializable, but the other VOs are JsonSerializable. It would be nice to delegate DocumentNumber's JSON serialisation to itself, in which case you can just write `'document_number' => $this->documentNumber,`.